services/alerts/src/__tests__/useAlerts.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { ReactNode } from 'react'
+import { renderHook } from '@testing-library/react-hooks'
+import { AlertsProvider, useAlerts } from '../index'
+
+const wrapper = ({ children }: { children?: ReactNode }) => (
+    <AlertsProvider>{children}</AlertsProvider>
+)
+
+describe('useAlerts', () => {
```

Comment:
Should we also test whether the alerts returned by the hook are the same that the alertsprovider provides?

Reply:
I don't think we need to do that. We are effectively asserting that the end result is correct. We know which alerts we should encounter and what they should look like. And we have already asserted that we are finding the correct ones.